simphony/cuds/lattice.py

```diff
@@ -67,6 +69,43 @@ def origin(self):
         return self._origin
 
     def get_node(self, index):
+        pass
```

Comment:
First of all, I think that the ABCLattice interface should include

@abstractproperty
    def name(self):

or even

@property
    def name(self):

and the same for all attributes listed in the docstings of ABCLattice (see https://docs.python.org/2/library/abc.html for abstract properties and related get/set-methods). Also, ABCLattice should probably include

 def __init__(self, name, type, base_vect, size, origin):

whether it is implemented or not (depending on @abstractproperty vs @property).

@tuopuu: do we really need the MetaLattice (or BaseLattice) class if we define a "full" interface for ABCLattice?
That is, are you really instantiating MetaLattice somewhere (in order to have an empty lattice)?